core/utils/style.ts

```diff
@@ -134,6 +106,9 @@ export function getComputedStyle(element: Element, property: string): string {
  * @alias Blockly.utils.style.getCascadedStyle
  */
 export function getCascadedStyle(element: Element, style: string): string {
+  deprecation.warn(
+      'Blockly.utils.style.getCascadedStyle', 'version 9.0.0',
+      'version 10.0.0');
```

Comment:
I like this clear approach, but I note that it doesn't makie any promises about how long the deprecated function will be available for.  For example, though unlikely it's not inconceivable that a serious security bug could force a major version bump the day after 9.0 is release.

Given that:
* we don't think this is used externally, and
* we're not _obliged_ to remove deprecated features

I think that's fine, but perhaps be careful about adopting this style more generally until we are reasonably sure we can set and stick to a schedule for major version bumps.